doc-surrealdb_versioned_docs/version-latest/cli/start.mdx

```diff
@@ -92,6 +92,14 @@ __BEFORE YOU START :__ Make sure you’ve [installed SurrealDB](/docs/surrealdb/
 
 ## Positional argument
 
+In the `surreal start` command, the path argument is used to specify the location of the database. If no argument is given, the default of `memory` for non-persistent storage in memory is assumed. 
+
+When using a path argument with SurrealDB, we recommend not using the `file://` prefix. Instead, please use [`surrealkv://`](/docs/surrealdb/cli/start#surrealkv-beta) or [`rocksdb://`](/docs/surrealdb/cli/start#surrealkv-beta) as the path argument. If you are using the `file://` prefix, you will see the following warning:
```

Comment:
Maybe a bit different phrasing to make it clear that this is just a powerless warning at the moment:

you will see the following warning -> the database will start with the following warning